app/assets/stylesheets/common.css.scss

```diff
@@ -34,7 +34,7 @@
 
 .btn-purple {
   color: white;
-  background-color: $purple;
+  background-color: rgba(65,50,107, .9);
```

Comment:
Thanks so much! It is the same color. I don't know much about sass yet, but it looks way less finicky.